src/HMRC.ESFA.Levy.Api.UnitTests/ApprenticeshipLevyApiClientTests.cs

```diff
@@ -63,12 +67,48 @@ public class ApprenticeshipLevyApiClientTests
             var client = new ApprenticeshipLevyApiClient(httpClient);
 
             // Act
-            //var declarations = await client.GetLevyDeclarations(expected.EmpRef);
             var ex = Assert.ThrowsAsync<ApiHttpException>(() => client.GetEmployerLevyDeclarations(expected.EmpRef));
 
             // Assert
             mockHttp.VerifyNoOutstandingExpectation();
             Assert.AreEqual(404, ex.HttpCode);
         }
+        
+
+        [Test]
+        public async Task ShouldGetLevyDeclarationsWithDeclarationTypes()
+        {
+            // Arrange
+            var expectedDeclarations = new List<Declaration>
+            {
+                new Declaration(),
+                new Declaration()
+            };
+            var expected = new LevyDeclarations
+            {
+                EmpRef = "000/AA00000",
+                Declarations = expectedDeclarations
+            };
```

Comment:
we should be mocking the variable 'var expected' in line 87 to live like response from hmrc.
`var expected = "{\"empref\": \"222/AA00002\",\"declarations\": [{\"id\": \"22200000212\",\"submissionId\": 0,\"dateCeased\": \"0001-01-01T00:00:00\",\"inactiveFrom\": \"0001-01-01T00:00:00\",\"inactiveTo\": \"0001-01-01T00:00:00\",\"noPaymentForPeriod\": false,\"submissionTime\": \"2018-04-15T12:00:00.000\",\"payrollPeriod\": {\"year\": \"17-18\",\"month\": 12},\"levyDueYTD\": 0.0,\"levyAllowanceForFullYear\": 15000.0}]}";`